airbnb/src/components/Calendar/CalendarBody.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+import CalendarDates from './CalendarDates';
+
+function CalendarBody(props) {
+  const { totalDate } = props;
+
+  const lastDate = totalDate.indexOf(1);
+  const firstDate = totalDate.indexOf(1, 7);
+
+  return (
+    <Form>
+      {totalDate.map((date, idx) => {
+        return (
+          <CalendarDates
+            key={date}
+            idx={idx}
+            lastDate={lastDate}
+            firstDate={firstDate}
+            date={date}
+          ></CalendarDates>
```

Comment:
자식이 없으면 바로 self closing 해주는게 어떨까요?
```suggestion
          />
```